lib/perl/Genome/Config/AnalysisProject/Command/View.pm

```diff
@@ -371,6 +372,35 @@ sub _color_status {
     return $self->_colorize_text_by_map($text, $text, %STATUS_COLORS);
 }
 
+sub _write_environment_config {
+    my ($self, $handle) = @_;
+
+    return if $self->config eq 'quiet';
+
+    my $environment_config = $self->analysis_project->environment_config_dir;
+    if ($environment_config) {
+        $self->_write_pairs_line($handle, 'Environment config', $environment_config);
+
+        if ($self->config eq 'verbose' or $self->config eq 'parsed') {
+            $self->_write_environment_config_file($handle, $environment_config);
+        }
+
+        print $handle "\n";
+    }
+}
+
+sub _write_environment_config_file {
+    my ($self, $handle, $environment_config) = @_;
+
+    my $f = File::Spec->join($environment_config, Genome::Config::config_subpath);
+    unless (-r $f) {
+        $self->warning_message('Could not read environment config <%s>.', $f);
```

Comment:
Since read_file calls validate_file_for reading, could this be written as a try catch?

use Try::Tiny 'try';
my $content = try {
  Genome::Sys->read_file($f)
}
catch {
   $self->warning_message('Could not read environment config <%s>.', $f);
}
...